docs/DeveloperGuide.md

```diff
@@ -374,29 +371,29 @@ Step 5. CONNECTIONS will display a detailed help message on the usage of `add` c
 
 Shows a list of contacts with upcoming birthdays. This list of birthday reminders is displayed to the user though the `UI`, specifically in `BirthdayReminderListPanel`. Each birthday is displayed as a `BirthdayReminderCard`.
 The list of birthdays is generated in the `ModelManager`, which implements the following functions:
-* `ModelManager#getBirthdayReminderList` returns an `ObservableList<Person>` that is ordered according to upcoming birthdays.
+* `ModelManager#getBirthdayReminderList` returns an `ObservableList<Person> birthdayReminders` that is ordered according to upcoming birthdays.
 
-Given below is an example usage scenario and how the Help mechanism behaves at each step.
+Given below is an example usage scenario and how the birthday reminders mechanism behaves at each step.
 
-Step 1. On app startup sort contacts with birthday by birth month and day only into a list of contacts. Birthdays that are one day away are coloured green while birthdays that within one week are coloured blue.
+Step 1. On app startup sort contacts with birthday by birth month and day only. Contacts with birthdays that fall on the present day are coloured green while birthdays that within one week are coloured blue. Additionally, a customised message with each contact's phone number and age is displayed in the birthday reminder list.
 
 Step 2. The first contact in the birthday reminder list will have the next birth month and day with respect 
 to current day.
 
 Step 3. The rest of the list with birthday after this first contact will be displayed in sorted order. This list is displayed in CONNECTIONS' `UI` in `BirthdayReminderListPanel`.
 
-Step 4. The user decides to scroll through the `BirthdayReminderListPanel`. Once at the end of the list (at contact with latest birthday), cycle back to the contact with the 
-earliest birthday and display remaining contacts in sorted order. 
+Step 4. The user decides to scroll through the `BirthdayReminderListPanel`. After the contact with the latest birthday is displayed, the contact with the 
+earliest birthday and all remaining undisplayed contacts with a birthday will be displayed in sorted order. 
```

Comment:
Can be in one line with the previous line. Maybe saying that birthday is sorted by day and month in chronological order is better?

Reply:
Ok thanks, just push a change hopefully its clearer, can help see if acceptable?